src/fsharp/FSharp.Compiler.Compilation/Extensions.fs

```diff
@@ -0,0 +1,98 @@
+﻿namespace FSharp.Compiler.Compilation
```

Comment:
Since this is getting duplicated here, would it make sense to have this live in a more common location?